ASWF/CLA-corporate.md

```diff
@@ -0,0 +1,58 @@
+Corporate Contributor License Agreement ("Agreement")
+
+Thank you for your interest in the Open Shading Language Project a Series of
+LF Projects, LLC (hereinafter "Project"). In order to clarify the
+intellectual property licenses granted with Contributions from any corporate
+entity to the Project, LF Projects, LLC ("LF Projects") is required to have
+a Corporate Contributor License Agreement (CCLA) on file that has been
+signed by each contributing corporation.
```

Comment:
This section should read....

```
Thank you for your interest in the Open Shading Language Project (hereinafter
 "Project"). In order to clarify the intellectual property licenses granted with 
Contributions from any corporate entity to the Open Shading Language Project 
a Series of LF Projects, LLC ("LF Projects") is required to have
a Corporate Contributor License Agreement (CCLA) on file that has been
signed by each contributing corporation.
```

Reply:
I pushed an update implementing these instructions.

Reply:
Perfect - thanks! We are getting in place these agreements for EasyCLA now.

Reply:
Ok - you're gonna kill me - but let's revert back to what you originally provided to make this consistent with other projects. Sorry :(